### PR TITLE
Added ospfv3 network rule

### PR DESCRIPTION
**Pull request by vvikramb**
Added ospfv3 network rule

### CODE REVIEW

File: juniper_official/vpn/l3vpn/l3vpn-proto-ospf3-nw.rule [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+                enumerate Attempt as -25;
+                enumerate Init as -20;
+                enumerate 2-Way as -15;
+                enumerate Exstart as -10;
+                enumerate Exchange as -5;
+                enumerate Loading as 0;
+                enumerate Full as 1;
+            }
+            trigger l3vpn-pe-interface-state {
+                frequency 1offset;
```
Comment (gaddevijay) [inline juniper_official/vpn/l3vpn/l3vpn-proto-ospf3-nw.rule:89]: alert-type needs to be set

File: juniper_official/vpn/l3vpn/l3vpn-proto-ospf3-nw.rule [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+                type string;
+                description "VRF name";
+            }
+            field vpn-state {
+                reference {
+                    path "/device-group[device-group-name={{pe-device-group}}]/device[device-id={{pe-device-name}}]/topic[topic-name='routing.ospf']/rule[rule-name=check-ospf3-neighbor-state]/field[neighbor-address='{{customer-neighbor-address}}']/ospf-neighbor-state";
+                    data-if-missing {
+                        value DOWN;
+                    }
+                    time-range 185s;
```
Comment (gaddevijay) [inline juniper_official/vpn/l3vpn/l3vpn-proto-ospf3-nw.rule]: Can we use little bigger time range? 1.5 times?

File: juniper_official/vpn/l3vpn/l3vpn-proto-ospf3-nw.rule [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+            }
+            field instance-interface-name {
+                constant {
+                    value "{{pe-interface-name}}";
+                }
+                type string;
+                description "Routing instance interface name";
+            }
+            field instance-interface-status {
+                reference {
```
Comment (gaddevijay) [inline juniper_official/vpn/l3vpn/l3vpn-proto-ospf3-nw.rule:32]: Please set time range

**Review by gaddevijay — CHANGES_REQUESTED** [review]

File: juniper_official/vpn/l3vpn/l3vpn-proto-ospf3-nw.rule (hunk 1 above)
Comment (vvikramb) [reply juniper_official/vpn/l3vpn/l3vpn-proto-ospf3-nw.rule:89]: added alert type

File: juniper_official/vpn/l3vpn/l3vpn-proto-ospf3-nw.rule (hunk 2 above)
Comment (vvikramb) [reply juniper_official/vpn/l3vpn/l3vpn-proto-ospf3-nw.rule]: made it 278s

File: juniper_official/vpn/l3vpn/l3vpn-proto-ospf3-nw.rule (hunk 3 above)
Comment (vvikramb) [reply juniper_official/vpn/l3vpn/l3vpn-proto-ospf3-nw.rule:32]: added 278s as time-range.

**Review by gaddevijay — APPROVED** [review]

**Pull request merged**